_data/simple-icons.json

```diff
@@ -9192,7 +9196,7 @@
         {
             "title": "Thymeleaf",
             "hex": "005F0F",
-            "source": "https://github.com/thymeleaf/thymeleaf-dist/tree/master/src/artwork/thymeleaf%202016"
+            "source": "https://github.com/thymeleaf/thymeleaf-dist/blob/main/src/site/images/thymeleaf.png"
```

Comment:
Should be https://github.com/thymeleaf/thymeleaf-dist/blob/a2e6b4a31c389a71f3f35f0decb48d7b1050df26/src/site/images/thymeleaf.png